installation.md

```diff
@@ -69,45 +69,47 @@ should work with any of them, though feature set and interface may vary.
 Here are some plugins we know work well with clangd:
 
 <details>
-<summary markdown="span">Vim</summary>
-[YouCompleteMe](https://valloric.github.io/YouCompleteMe/) can be installed with
-clangd support. **This is not on by default**, you must install it with
-`install.py --clangd-completer`.
+<summary markdown="span">Vim/Neovim</summary>
```

Comment:
Can this just be "Vim" in the heading? It's nice if the headings are easy to scan over (for people looking for other editors), and I don't think anyone using Neovim is likely to be confused, especially when we mention it in the next sentence.